server/executor/replacer/replacer.go

```diff
@@ -0,0 +1,25 @@
+package replacer
+
+import (
+	"fmt"
+
+	"github.com/kubeshop/tracetest/server/model"
+)
+
+var defaultInjector *Injector = nil
```

Comment:
Do we really need this to be a reference? It can be made a direct instance:

```suggestion
var defaultInjector Injector = NewInjector()
```